### PR TITLE
DOCUMENTATION: Update README file

### LINKED ISSUE #6: DOCUMENTATION: Update README file
closes #6

### PR DESCRIPTION
**Pull request by NaynaMallick**
closes #6

### CODE REVIEW

**Comment by ShriHumrudha** [comment]
Do upload the image too.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Do upload the image too.

**Comment by NaynaMallick** [comment]
Uploaded image file.

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-# LatestVersion.Services
\ No newline at end of file
+# .Services
+[![.NET](https://github.com/Mums-Who-Code/ArtGallery.Services/actions/workflows/dotnet.yml/badge.svg)](https://github.com/Mums-Who-Code/ArtGallery.Services/actions/workflows/dotnet.yml)
+[![The Standard - COMPLIANT](https://img.shields.io/badge/The_Standard-COMPLIANT-2ea44f)](https://github.com/hassanhabib/The-Standard)
+
+# ArtGallery.Services
+
+<p align="center">
+  <img width="25%" height="25%" src="https://github.com/Mums-Who-Code/Random.Services/blob/main/artgallery.jpg">
```
Comment (ShriHumrudha) [inline README.md]: Update the path

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by Shrabanac — APPROVED** [review]

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**